tools/perf/ResultsComparer/README.md

```diff
@@ -18,33 +32,190 @@ It can be used to compare:
+| SerializationBaselineTests.SerializationBenchmarks.WriteODataSyncArrayPoolFile(d | 1.2182187163088392 |      24971900.00 |      20498700.00 |         |
+| SerializationBaselineTests.SerializationBenchmarks.WriteODataSyncArrayPool(dataS |  1.218181364804361 |     251591850.00 |     206530700.00 |         |
+| SerializationBaselineTests.SerializationBenchmarks.WriteODataSyncArrayPoolFile(d | 1.2181029952311364 |     243142250.00 |     199607300.00 |         |
+| SerializationBaselineTests.SerializationBenchmarks.WriteODataSyncFile(dataSize:  | 1.2158922478078782 |      26332700.00 |      21657100.00 |         |
+| SerializationBaselineTests.SerializationBenchmarks.WriteODataSyncFile(dataSize:  | 1.2127900327408094 |     129795700.00 |     107022400.00 |         |
+| SerializationBaselineTests.SerializationBenchmarks.WriteODataSyncArrayPool(dataS |  1.211470433519456 |      25103000.00 |      20721100.00 |         |
+| SerializationBaselineTests.SerializationBenchmarks.WriteODataSyncArrayPoolFile(d | 1.2045993443253906 |      26290200.00 |      21824850.00 |         |
+
+No New results for the provided threshold = 15% and noise filter = 0.3ns.
+
+No Missing results for the provided threshold = 15% and noise filter = 0.3ns.
+
+## Comparing Visual Studio .NET Object Allocation Tracker Profiler reports
+
+The tool currently supports comparing data generated by the [.NET Objec Allocation Tracker](https://devblogs.microsoft.com/visualstudio/net-object-allocation-tool-performance/) and [Memory Usage](https://docs.microsoft.com/en-us/visualstudio/profiling/dotnet-alloc-tool?view=vs-2022) in the Visual Studio Profiler.
+
+We currently don't support reading the actual `.diagsession` files generated by the profiler, and Visual Studio doesn't provide an export feature either. So to get a report this tool can read, you will have to manually copy it into a text file. To do so:
+- Open the tab of the view you want to compare (e.g. `Allocations` or `Functions`)
+- Select the rows you want to copy. To select the entire table, click the first row then scroll down and click the last row while holding down `Shift+A`
+- Right-click then click **Copy**
+- Open/Create an empty text file and paste the data there and save the file
+- Do this for both the base and diff reports
+
+![Copy VS Profiler Report](./assets/VSProfilerAllocationsCopyData.jpeg)
+
+![Pasted VS Profiler Text Data](./assets/VSProfilerAllocationsCopiedTextData.jpeg)
+
+With the files ready, you can run the command. Currently we support data from the **Allocations** tab and **Functions** tab:
+
+### .NET Object Allocation Tracker - Type Allocations
+
+This is the **Allocations** tab of the profiler.This report shows the number and size of allocations for each data type.
+For this type of report, use **`vsAllocs`** as the `--type` option:
+
+```
+dotnet run -- --type vsAllocs --base C:\reports\AllocationsReportBefore.txt --diff C:\reports\AllocationsReportAfter.txt
```

Comment:
```suggestion
dotnet run --type vsAllocs --base C:\reports\AllocationsReportBefore.txt --diff C:\reports\AllocationsReportAfter.txt
```
Or is the `--` after `dotnet run` and before `--type vsAllocs` required?

Reply:
Yeah, it's used to separate the arguments passed to `dotnet run` and those passed to the program being executed by `dotnet run`